src/stable/test/index.ts

```diff
@@ -39,50 +39,60 @@ export function runTests(tests: Test): void {
     (shouldRunTests === true ? describe : describe.skip)(
         `correctness tests`,
         () => {
-            (shouldCheckMemoryStateAfterTests === true ? please : please.skip)(
-                'snapshot the canister memory size and heap allocation for all canisters before correctness tests',
-                async () => {
+            if (shouldCheckMemoryStateAfterTests === true) {
+                beforeAll(async () => {
```

Comment:
I want the beforeAll and afterAll to always run no matter what, please keep the original if statements simply inside of the beforeAll and afterAll.